tests/data/sm-os_80hdx.json

```diff
@@ -0,0 +1 @@
+null
```

Comment:
:)

Reply:
*slaps wrist*

I find if I'm running lnms dev:simulate sometimes, it seems to lead to empty json files. I really should have checked though :)